composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": "~5.3",
+        "php": ">=5.3",
```

Comment:
@olivier34000 i would prefere the following: fixed it allready within https://github.com/saxulum/saxulum-doctrine-mongodb-provider

```{.json}
 "php": "~7.0|~5.3"
```